vktrace/vktrace_layer/vktrace_lib_trace.cpp

```diff
@@ -953,6 +953,13 @@ VKTRACER_EXPORT VKAPI_ATTR VkResult VKAPI_CALL __HOOKED_vkCreateDevice(VkPhysica
     // remove the loader extended createInfo structure
     VkDeviceCreateInfo localCreateInfo;
     memcpy(&localCreateInfo, pCreateInfo, sizeof(localCreateInfo));
+
```

Comment:
The lists of extensions or layers don't appear to be modified. I've updated the fix.